pootle/templates/languages/admin/characters.html

```diff
@@ -0,0 +1,22 @@
+{% extends "languages/base.html" %}
+
+{% load assets i18n %}
+
+{% block title %}{% trans "Special Characters" %} | {{ block.super }}{% endblock %}
+
+{% block content %}
+<div class="settings-container centered" lang="{{ LANGUAGE_CODE }}">
+  <p class="info">{% trans "Below you can edit the special characters for this language." %}</p>
```

Comment:
Not sure how helpful this text is, although I admit the page itself feels completely naked too. I'd go on and align this page's look & feel with the user settings page.

![specialchars](https://cloud.githubusercontent.com/assets/16768/10878750/131edf5e-8150-11e5-9eb7-b0f6de7b7f4c.png)